### PR TITLE
Feature/reset time lists in farmer harvester log file

### PR DESCRIPTION
**Pull request by tnt-manu**

### CODE REVIEW

File: chia_tea/chia_watchdog/logfile/line_checks.py [hunk 1]
```diff
@@ -123,7 +123,7 @@ def apply(
             timestamp_dt)
         harvester_info.last_update = timestamp_dt
 
-        harvester_info.is_connected = True
+        #harvester_info.is_connected = True  ## this meakes no sense ?
```
Comment (tnt-codie) [inline chia_tea/chia_watchdog/logfile/line_checks.py]: the remove it

File: chia_tea/chia_watchdog/logfile/line_checks.py [hunk 2]
```diff
@@ -154,6 +154,7 @@ def apply(
             ip_address
         )
         harvester_info.is_connected = True
+        harvester_info.reset_send_recieve_lists()
```
Comment (tnt-codie) [inline chia_tea/chia_watchdog/logfile/line_checks.py:157]:
```suggestion
        harvester_info.reset_send_receive_lists()
```

File: chia_tea/chia_watchdog/logfile/line_checks.py (hunk 1 above)
Comment (tnt-manu) [reply chia_tea/chia_watchdog/logfile/line_checks.py]: done

**Comment by tnt-manu** [comment]
where as the n_overdue_responses should be good for now, still in question is the logic for the notification.
As for now, timeout notifications only happes with db data - not sure if that is precise / valid enough in case of a cable pull without a proper disconnect - doubt it. Maybe test it on the real log file or update the db values based on the logfile checks - what do you think @tnt-codie ?

**Review by tnt-codie — APPROVED** [review]
LGTM now. Just fix the linting stuff and we can merge.

**Pull request merged**